### PR TITLE
Bump to 1.32.5 (updated node_preamble.dart)

### PR DESCRIPTION
**Pull request by Mstrodl**
Similar to #1028, pulls mbullington/node_preamble.dart#21

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 1.32.5
+
+* Use an updated version of `node_preamble` when compiling to JS.
```
Comment (nex3) [inline CHANGELOG.md:3]: Generally, we try to keep changelog entries focused on the user's perspective. What does the latest version of `node_preamble` allow/fix that older versions did not?

**Review by nex3 — CHANGES_REQUESTED** [review]

**Comment by Mstrodl** [comment]
@nex3 I've updated the changelog as requested

**Comment by Mstrodl** [comment]
@stof See the old PR, I was told not to change it because it still fits within the semver.

**Comment by Mstrodl** [comment]
Ironically however this doesn't even fix my issue because node_preamble.dart didn't actually have it's version number changed... Oops.

**Comment by stof** [comment]
but if the fix requires the new version of the library, bumping the min version of the constraint would make sense to be sure that `pub` cannot select the older buggy version. Otherwise, the fix is not enforced.

**Comment by Mstrodl** [comment]
@stof I agree with what you're saying, I'm just explaining why I didn't bump it (I did try the first time, see #1012 .)

**Comment by stof** [comment]
well, then the issue is not actually fixed, because the fix you want to bring in is not yet released on https://pub.dev/packages/node_preamble

**Comment by Mstrodl** [comment]
@stof Correct! I have opened a pull request on the node_preamble repository because I forgot: mbullington/node_preamble.dart#23

**Pull request merged**